Flutter/cartesian-charts/methods.md

```diff
@@ -459,6 +459,167 @@ The [`hide`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/char
 {% endhighlight %}
 {% endtabs %}
 
+### Events in trackballBehavior
+
+Integrated the following methods for handling pointer and gesture events and allowing customizations for various pointer events such as long-press, tap, double-tap, pointer enter, and exit.
+
+* `handleEvent` - Specifies to customize the necessary pointer events.
+* `handleLongPressStart` - Specifies to customize the pointer when a long-press begins.
+* `handleLongPressMoveUpdate` - Specifies to customize the pointer during movement after a long-press.
+* `handleLongPressEnd` - Specifies to customize the pointer when the pointer stops contacting the screen after a long-press.
+* `handleTapDown` - Specifies to customize the pointer when a tap has contacted the screen once.
+* `handleTapUp` - Specifies to customize the pointer when it has stopped contacting the screen after a tap.
+* `handleDoubleTap` - Specifies to customize the pointer when a tap has contacted the screen twice.
+* `handlePointerEnter` -Specifies to customize the pointer when the mouse enter on the screen.
+* `handlePointerExit` - Specifies to customize the pointer when the mouse exit on the screen.
+
+{% tabs %}
+{% highlight dart %} 
+
+    class CustomTrackballEvent extends TrackballBehavior {
+      @override
+      bool get enable => true;
+
+      @override
+      ActivationMode get activationMode => super.activationMode;
+
+      // Stops the behavior of mouse pointer move, cancel, hover and up events.
```

Comment:
Disable the behavior of pointer move,

Reply:
Used this "Disable the behavior of the pointer move"